### DISCUSSION TITLE
Why the reality must use TLS1.3 + H2?

### DISCUSSION THREAD

**Discussion by caozhenbiao** [XTLS/Xray-core#2382] [General]
Why the reality must use TLS1.3 + H2?

**Comment by chika0801** [comment]
https://github.com/XTLS/Xray-core/issues/1891#issuecomment-1495439413

You need to be able to read the code yourself for specifics. Here's some previously gathered intel. Maybe you can wait until the author writes a detailed description. (Plans for next year)
